docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -284,10 +282,10 @@ Release date: May. 31, 2018
 - Stabilized evas rendering mechanism.
 
 ### Known Issues
-- CBHM (Clip Board History Manager) is not work properly
-- Ector supports only basic functionalities compared to Cairo
-- SVG morphing animation supports only in EDC
-- Elementary focus manager is disabled (legacy focus mechanism is applied)
+- CBHM (Clip Board History Manager) is not work properly.
+- Ector supports only basic functionalities compared to Cairo.
+- SVG morphing animation supports only in EDC.
+- Elementary focus manager is disabled (legacy focus mechanism is applied).

```

Comment:
If legacy focus is a name then it can be capitalized.

Reply:
will accept